tests/Makefile

```diff
@@ -34,7 +34,11 @@ DIRS += json
 DIRS += conf
 DIRS += nbio
 DIRS += thread
+
+ifndef MYST_RELEASE
```

Comment:
Now `MYST_RELEASE` build supports debugging if the the TEE-debug-mode is true. Should gdb and dotnet-sos tests be enabled with `MYST_RELEASE` build?

Reply:
Good find. I reenabled those tests now that we have auto-detection of debug mode.